uk.jnthn.backgammony.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "uk.jnthn.backgammony",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
```

Comment:
Can you check if `org.freedesktop.Sdk.Extension.dmd//19.08` still works? I think it's happy coincidence it works now as 19.08 is the only branch for this extension.

Reply:
@barthalion sorry would you be able to expand on what you mean by this?

Reply:
@barthalion The extension point in Gnome SDK 3.34 explicitly sets it's version to `19.08`:
```ini
[Runtime]
name = org.gnome.Sdk
runtime = org.gnome.Platform/x86_64/3.34
sdk = org.gnome.Sdk/x86_64/3.34

...

[Extension org.freedesktop.Sdk.Extension]
subdirectories = true
directory = lib/sdk
no-autodownload = true
add-ld-path = lib
version = 19.08
```

Reply:
@gasinvein I don't think it matters at all but let's worry in 6 months.